octo-core/lib/octocore/models/enterprise/funnel_tracker.rb

```diff
@@ -0,0 +1,19 @@
+require 'cequel'
+
+# A model for tracking the user web flow
```

Comment:
This comment should come before `class Funnel..` class definition. Else it would be generated against Octo module.